packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js

```diff
@@ -199,8 +199,12 @@ export class EvidenceMap {
 						this.lastSelectedLayer = layer;
 					}
 					onclick(feature);
+				});
+				layer.on('dblclick', () => {
+					//handles map drilldowns
 					if (link) {
 						window.location.href = link;
+						this.map.doubleClickZoom.disable();
```

Comment:
It seems a little weird to disable the `doubleClickZoom` only after the user has double clicked the map. Maybe this should happen when the map is created if they provide links?